Source/FieldSolver/SpectralSolver/SpectralFieldDataRZ.cpp

```diff
@@ -6,9 +6,10 @@
  */
 #include "SpectralFieldDataRZ.H"
 
-#include "WarpX.H"
```

Comment:
Still needed for `WarpX::getCosts(lev);`:
```suggestion
#include "WarpX.H"
```